policyengine_canada/parameters/gov/cra/tax/income/credits/gst_credit/reduction_multiplier.yaml

```diff
@@ -0,0 +1,12 @@
+description: GST credit reduction multiplier
```

Comment:
Let's use a scale parameter for the reduction, like https://github.com/PolicyEngine/policyengine-canada/blob/master/policyengine_canada/parameters/gov/cra/benefits/cwb/phase_out/base/single.yaml

This removes the need for a separate `reduction_threshold.yaml` file.